modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -7,12 +7,12 @@ endif::[]
 image::configure-service-mesh-breadcrumb.png[]
 
 
-[[adapter-overview]]
-== Overview
+// [[adapter-overview]]
+// == Overview
```

Comment:
Why are you commeting out titles and anchors? Are you using them somewhere else? 
If these are not necessary, I'd suggest removing them.

Reply:
I don't see them being used anywhere. I just want to check I once. I'll remove these the release tomorrow.